src/core/Akka/Routing/RoutedActorCell.cs

```diff
@@ -160,14 +160,7 @@ public override void Start()
             if (RouterConfig is Pool)
             {
                 var pool = (Pool)RouterConfig;
-                // must not use pool.GetNrOfInstances(system) for old (not re-compiled) custom routers
-                // for binary backwards compatibility reasons
-                var deprecatedNrOfInstances = pool.NrOfInstances;
-
-                var nrOfRoutees = deprecatedNrOfInstances < 0
```

Comment:
any routers that are compiled without `NrOfInstances(ActorSystem)` can't run on 1.1 or later anyway.